doc/examples/transform/plot_ring_detector.py

```diff
@@ -0,0 +1,132 @@
+"""
+=============================
+Ridge Directed Ring Detection
+=============================
+
+Robust and highly performant ring detection algorithm for 3d particle tracking
+using 2d microscope imaging.
+
+Here's a quick description of the parameters. The explanations refer to the
+terms as introduced in [Afik 2015](https://www.nature.com/articles/srep13584)
```

Comment:
Looks like you're using Markdown syntax. In my experience, it should be reStructuredText.

Reply:
indeed :-)

Reply:
you can take a look at existing gallery scripts for examples of rst syntax.

Reply:
Sorry about that @mkcor @emmanuelle, but this is too raw for a review yet. :smile: I asked @clementkng to move the code from a notebook to this file, starting the documentation process. We'll take care of it :slightly_smiling_face: